k8s/conf/icmp-responder-nginx.yaml

```diff
@@ -0,0 +1,42 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+spec:
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        networkservicemesh.io/app: "icmp-responder"
+        networkservicemesh.io/impl: "icmp-responder"
+    spec:
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            - labelSelector:
+                matchExpressions:
+                  - key: networkservicemesh.io/app
+                    operator: In
+                    values:
+                      - icmp-responder
+                  - key: networkservicemesh.io/impl
+                    operator: In
+                    values:
+                      - icmp-responder
+              topologyKey: "kubernetes.io/hostname"
+      containers:
+        - name: icmp-responder-nse
```

Comment:
You may want to name this container "vpn-gateway".

Also, make the [networkService in icmp-responder-nse](https://github.com/ligato/networkservicemesh/blob/master/examples/cmd/icmp-responder-nse/nse.go#L35) configurable from an ENV variable, because we will need to advertise it as Network Service "secure-intranet-gateway"